content/docs/examples/multicluster/icp/index.md

```diff
@@ -0,0 +1,225 @@
+
+    [Configure cluster-1 with kubectl](//www.ibm.com/support/knowledgecenter/SSBS6K_2.1.0.3/manage_cluster/cfc_cli.html).
+
+    1.  Check the cluster status:
+
+        {{< text bash >}}
+        $ kubectl get nodes
+        $ kubectl get pods --all-namespaces
+        {{< /text >}}
+
+    1.  Repeat above steps to validate `cluster-2`.
+
+## Config Pod Communication Across IBM Cloud Private Clusters
+
+IBM Cloud Private uses Calico Node-to-Node Mesh by default to manage container networks. The BGP Client
+on each node distributes the IP router information to all nodes.
+
+To ensure pods can communicate across different clusters, you need to configure IP routers on all nodes
+in the cluster. You need two steps:
+
+1.  Add IP routers from `cluster-1` to `cluster-2`.
+
+1.  Add IP routers from `cluster-2` to `cluster-1`.
+
+You can check how to add IP routers from `cluster-1` to `cluster-2`. With Node-to-Node Mesh mode, each
+node will have IP routers connecting to peer nodes in the cluster. In this example, both cluster has
+three nodes.
+
+The `hosts` file for `cluster-1`:
+
+{{< text bash >}}
+9.111.255.21 gyliu-icp-1
+9.111.255.129 gyliu-icp-2
+9.111.255.29 gyliu-icp-3
+{{< /text >}}
+
+The `hosts` file for `cluster-2`:
+
+{{< text bash >}}
+9.111.255.152 gyliu-ubuntu-3
+9.111.255.155 gyliu-ubuntu-2
+9.111.255.77 gyliu-ubuntu-1
+{{< /text >}}
+
+1.  Obtain routing information on all nodes in `cluster-1` with the command `ip route | grep bird`.
+
+    {{< text bash >}}
+    root@gyliu-icp-1:~# ip route | grep bird
+    10.1.43.0/26 via 9.111.255.29 dev tunl0 proto bird onlink
+    10.1.158.192/26 via 9.111.255.129 dev tunl0 proto bird onlink
+    blackhole 10.1.198.128/26 proto bird
+    {{< /text >}}
+
+    {{< text bash >}}
+    root@gyliu-icp-2:~# ip route | grep bird
+    10.1.43.0/26 via 9.111.255.29 dev tunl0  proto bird onlink
+    blackhole 10.1.158.192/26  proto bird
+    {{< /text >}}
+
+    {{< text bash >}}
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    root@gyliu-icp-3:~# ip route | grep bird
+    blackhole 10.1.43.0/26  proto bird
+    10.1.158.192/26 via 9.111.255.129 dev tunl0  proto bird onlink
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    {{< /text >}}
+
+1.  There are three IP routers total for those three nodes in `cluster-1`.
+
+    {{< text bash >}}
+    10.1.158.192/26 via 9.111.255.129 dev tunl0  proto bird onlink
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    10.1.43.0/26 via 9.111.255.29 dev tunl0  proto bird onlink
+    {{< /text >}}
+
+1.  Add those three IP routers to all nodes in `cluster-2` by the command to follows:
+
+    {{< text bash >}}
+    $ ip route add 10.1.158.192/26 via 9.111.255.129 
+    $ ip route add 10.1.198.128/26 via 9.111.255.21
+    $ ip route add 10.1.43.0/26 via 9.111.255.29
+    {{< /text >}}
+
+1.  You can use same steps to add all IP routers from `cluster-2` to `cluster-1`, as well. After configuration
```

Comment:
use same steps -> use the same steps.  Remove , as well.

Reply:
Done